documentacion/como_empezar/README.md

```diff
@@ -62,7 +62,7 @@ gem install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Ruby](https://github.com/TransbankDevelopers/transbank-sdk-ruby#instalaci%C3%B3n) para mas opciones de instalación.
```

Comment:
...para más opciones...

Reply:
Not related to this PR, it could be a good idea if you submit a PR with spellcheking.